backends/cuda-shared/ceed-cuda-shared-basis.c

```diff
@@ -781,7 +781,8 @@ int CeedBasisApplyTensor_Cuda_shared(CeedBasis basis, const CeedInt nelem,
                                         interpargs); CeedChk(ierr);
     } else if (dim == 2) {
       const CeedInt optElems[7] = {0,32,8,6,4,2,8};
-      CeedInt elemsPerBlock = Q1d < 7 ? optElems[Q1d]/ncomp : 1;
+      // elemsPerBlock must be at least 1
+      CeedInt elemsPerBlock = (Q1d < 7 ? optElems[Q1d]/ncomp : 1) || 1;
```

Comment:
I don't understand how this can work, `||` returns a `bool`, so all this is doing is returning `1`, or am I missing something?

Reply:
I'm using short circuiting. I could also write this as a ternary, but it would be longer.

Reply:
That's what I get for trying to be clever. `||` indeed returns a bool which is converted to `1`. Updated.